src/components/StaticAppBar/StaticAppBar.react.js

```diff
@@ -53,6 +53,7 @@ class StaticAppBar extends Component {
       showOptions: false,
       showAdmin: false,
       anchorEl: null,
+      num: new Date().getTime(),
```

Comment:
Please name it to `timestamp`, `num` is unclear.

Reply:
Sure, on it.